pkg/webui/console/views/overview/index.js

```diff
@@ -247,6 +248,23 @@ export default class Overview extends React.Component {
                   component="h2"
                 />
               )}
+              <Message
+                className={style.getStarted}
+                content={m.needHelp}
+                values={{
+                  documentationLink: (
+                    <Link.Anchor secondary href={documentationBaseUrl}>
```

Comment:
```suggestion
                    <Link.Anchor secondary href={documentationBaseUrl} target="_blank">
```

We want to open a new tab for the user